modules/lightning_features/lightning_media/src/BundleResolverInterface.php

```diff
@@ -23,4 +23,12 @@
    */
   public function getBundle($input);
 
+  /**
+   * Returns all possible bundles for the field type(s) this plugin supports.
+   *
+   * @return MediaBundleInterface[]
+   *   Applicable media bundles, keyed by ID.
+   */
+  public function getPossibleBundles();
```

Comment:
Can we have a base implementation of this method in BundleResolverBase which simply returns all bundles, unconditionally?